### DISCUSSION TITLE
能否隐藏 textArea 的滚动条? 现在会导致 print 时内容显示不全

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#45069] [Q&A]
<img width="1249" alt="image" src="https://github.com/ant-design/ant-design/assets/124666577/ba5ea54c-93a5-423d-9a89-66fa93d61109">

这是 mac 
<img width="1011" alt="image" src="https://github.com/ant-design/ant-design/assets/124666577/3b7af858-f883-4693-b261-fa0c85b0df23">

这是 windows ,滚动条不仅影响美观 而且会导致打印内容不全
<img width="903" alt="image" src="https://github.com/ant-design/ant-design/assets/124666577/3ed2b857-f9d8-4fbb-b51a-b7d9a59f45b0">

**Comment by zombieJ** [comment]
`overflow: hidden` 或者 css 自定义滚动条

**Reply by vaynevayne** [reply]
不行, 内容还是不全,autoRows 是你们自己实现的吗? 好像得把滚动条高度考虑进去才行

**Comment by vaynevayne** [comment]
我的打印使用的是 react-to-print , 在一个 iframe 里打印, 当iframe 出现时,页面上的 textarea 的滚动条就会出现, 这不是因为 js 无法计算 iframe 内部的 textarea 的高度?
